### DISCUSSION TITLE
Just asking for asking's sake due to firefox's current state

### DISCUSSION THREAD

**Discussion by aahsnr-configs** [zen-browser/desktop#3391] [General]
For a while now, with each update firefox seems to degrading in quality. Webpages don't load properly, internet slows down quite significantly and many more issues, plus the firefox organization seems to be making changes internally. Since DoD in the states is trying to make google sell chrome to a 3rd party, google's deal with firefox where it makes google search engine the default might be trouble. That's where the mozilla foundation might have significant financial problems.

With all that being said ( and it's all speculative, of course), I was wondering if any of firefox's degrading quality and their possible financial problem is gonna affect the development of the zen-browser team. Truth be told, because of the issues I have been having with firefox, I have switched to zen browser completely and it's literally the bomb and I would like to extend my gratitude to the team. But since zen-browser is a firefox derivative, do you guys think you might face any problems with zen browser's development in the future if firefox itself faced more major issues.

The reason I am asking all these stuff is because I mainly use wayland in linux full-time, and firefox and its derivatives tend to fair better in terms of scaling or what not than their chromium counterparts. Plus, firefox's security and privacy benefits tend to trickle down to its derivatives. And I appreciate that the zen browser's team to make that a priority. And not to forget that zen browser, in my experience, is significantly faster than any chromium based browsers I have used so far. I mean, before firefox had their issues, webpages loaded less faster than chromium based browsers but using firefox was still usable. 

To summarize all these rambling, how are you guys fairing so far in your development process of zen browser. I usually don't find any issues with your current release as well as past releases, so I am guessing development is going smoothly. But are you still at all worried about the browser's future? 

P.S. I apologize for the long rambling in advance, and thanks in advance. The other reason for the rambling is because I deeply care about the project since it has helped me so much personally.

Regards,
Your average browser user

**Comment by rubaboo** [comment]
You think Firefox is degrading in quality, and you don't find any issues with Zen releases? My experience has been completely opposite. I tried Zen and liked it, but gave up on it after seeing the deluge of requests for convoluted features satisfying some queer usage scenarios, and the team taking them on without any consideration for delivering solid well-thought-out and well-tested SANE set of features.

**Reply by youwen5** [reply]
Zen _is_ firefox with a skin. There aren't really any novel changes to the underlying Gecko engine or browser internals made by Zen, as of now. It's essentially just some configuration optimization on Firefox and a lot of UI patches. (I'm not trying to downplay the work of Zen's maintainers, just pointing out that its browser core is not meaningfully different from Firefox.)

Zen is not a "hard fork," it still has to keep up with Firefox releases to receive critical security updates and stay up to date with the web. Zen does not have nearly enough resources to maintain an entirely divergent fork of Firefox and provide their own security fixes and updates. So anything that Firefox breaks in their browser will also trickle down to Zen.

**Reply by adminy** [reply]
Same here, I've experienced the opposite so far. Firefox everything was working more or less without any issues. Zen is in your face about the updates and with every update its just gradually worse. I'm already thinking to go back to firefox.

I've seen web pages just not working entirely with firefox, I know what you mean when you say that, same page would just work in chrome for example. I believe those are react web pages that target only chrome browsers, taking advantage of that one feature react only has support in chrome for. I keep a spare chromium / brave instance around just in case.

**Reply by JSGRD22** [reply]
The quality is and will be picking up again in the last couple of updates as the developers focus more on stability and polishing.
